cmd/lncli/commands.go

```diff
@@ -146,6 +146,52 @@ func newAddress(ctx *cli.Context) error {
 	return nil
 }
 
+var estimateFeeCommand = cli.Command{
+	Name:      "estimatefee",
+	Category:  "On-chain",
+	Usage:     "Get fee estimates for sending bitcoin on-chain to multiple addresses.",
+	ArgsUsage: "send-json-string [--conf_target=N]",
```

Comment:
The json args are a _bit_ awkward here, as it can be difficult to manually craft them on the command line. Not sure what a better alternative is though given the restrictions at hand.

Reply:
I agree, but this was just blatantly copied from the `SendMany` command. 

We need to be able to specify a variable number of addresses, not sure if there's a better way to do this using this command line tool. What about expecting a single comma separated string `"<addr1>:<coins1>,<addr2>:<coins2>"` etc?

Reply:
Hmm, yeah I think that's an easier format to type out manually on the command line. Your call if you want to implement this now, or defer it to a follow up.